docs/notebooks/analysis/swept_langmuir/find_floating_potential.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
<p>Maybe change the vertical line color from orange? The fits are displayed in orange already.</p><p> </p><p>Also, one thing I found out about recently - xlim, ylim, xlabel and ylabel (and some others, but probably not ones here) could be wrapped into a single <code>ax.set</code>.  Try it out! It's less code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/889/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Good thing to note.  `ax.set` does not seem to be very robust though.  I can't even set font sizes with it.